go.mod

```diff
@@ -283,15 +284,20 @@ require (
 	k8s.io/gengo/v2 v2.0.0-20240911193312-2b36238f13e9 // indirect
 	k8s.io/kube-aggregator v0.32.2 // indirect
 	k8s.io/kubernetes v1.32.2 // indirect
-	nhooyr.io/websocket v1.8.7 // indirect
 	sigs.k8s.io/json v0.0.0-20241010143419-9aa6b5e7a4b3 // indirect
 	sigs.k8s.io/kustomize/api v0.18.0 // indirect
 	sigs.k8s.io/kustomize/kyaml v0.18.1 // indirect
 )
 
 replace (
+	// andrii-korotkov-verkada replaces are for libraries deps updates and are hopefully temporary until lib maintainers accept the patches
 	github.com/golang/protobuf => github.com/golang/protobuf v1.5.4
+	github.com/google/go-jsonnet => github.com/andrii-korotkov-verkada/go-jsonnet v0.0.0-20250308155707-03f98b787f0e
 	github.com/grpc-ecosystem/grpc-gateway => github.com/grpc-ecosystem/grpc-gateway v1.16.0
+	github.com/improbable-eng/grpc-web => github.com/andrii-korotkov-verkada/grpc-web v0.0.0-20250308221419-d37df84c6882
+	github.com/olekukonko/tablewriter => github.com/andrii-korotkov-verkada/tablewriter v0.0.0-20250308155854-8571ed54aa92
+	github.com/r3labs/diff/v3 => github.com/andrii-korotkov-verkada/diff/v3 v3.0.0-20250308155935-ef18f9606732
+	github.com/soheilhy/cmux => github.com/andrii-korotkov-verkada/cmux v0.0.0-20250308160011-2ab3fff3705c
```

Comment:
I really don't think we should do this. imo using a fork of a lightly-maintained library is worse than just using the lightly-maintained library.